### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by pconesa**

### CODE REVIEW

**Comment by pconesa** [comment]
This will send devel to master and publish the package to pypi automatically..
I leave it as draft. Feel free to merge it when content can be published.

File: xmipptomo/protocols/protocol_cltomo.py [hunk 1]
```diff
@@ -23,14 +23,13 @@
 # *  e-mail address 'scipion@cnb.csic.es'
 # *
 # **************************************************************************
+import os
```
Comment (EstrellaFG) [inline xmipptomo/protocols/protocol_cltomo.py:26]: Not sure if this protocol is properly working. This protocol was in xmipp3 plugin as a first try for subtomogram classification. When plugin xmipptomo was created, this protocol was migrated here, but I think for the moment is a 'work in progess in pause', maybe we can leave it just in devel and not release it @pconesa? (same with its viewer)

File: xmipptomo/protocols/protocol_subtomo_map_back.py [hunk 2]
```diff
@@ -26,16 +26,18 @@
 # *
 # **************************************************************************
 
-from pyworkflow.em import ALIGN_3D
-from pyworkflow.em.convert import ImageHandler
-import pyworkflow.em.metadata as md
-from pyworkflow.em.protocol import EMProtocol
+from os.path import basename
```
Comment (EstrellaFG) [inline xmipptomo/protocols/protocol_subtomo_map_back.py:29]: This protocol is being testing with the users with real (and complex) data, it seems the code is fine, but maybe soon we find something to fix.

File: xmipptomo/viewers/viewer_cltomo.py [hunk 3]
```diff
@@ -30,10 +30,9 @@
 
 import glob
 
-from pyworkflow.em import *
+from pwem.viewers import DataView
```
Comment (EstrellaFG) [inline xmipptomo/viewers/viewer_cltomo.py:33]: not sure if the protocol for this viewer is working

File: xmipptomo/protocols/protocol_subtomo_map_back.py (hunk 2 above)
Comment (EstrellaFG) [reply xmipptomo/protocols/protocol_subtomo_map_back.py:29]: This protocol is already working properly

**Pull request merged**